src/components/AdminSettings/TurnServer.vue

```diff
@@ -63,7 +64,7 @@
 			</template>
 		</NcButton>
 		<NcButton v-show="!loading"
```

Comment:
Add a `flex-shrink: 0` maybe, to make it square-ish?
![image](https://github.com/user-attachments/assets/61147699-0bb8-4142-94ce-4329df48d0cf)

Reply:
Nah, was caused by `align-items: stretch`

Reply:
Fixed